OriginateTheme/Scripts/ot_generator.py

```diff
@@ -785,9 +810,12 @@ def main(argv):
             sys.exit()
 
     # Create the fonts, colors and components output files.
-    generateUITypeClass(outputDirectory, 'OTFonts', sorted(fonts, key = lambda x: x.key), 'font', createFontGetter)
-    generateUITypeClass(outputDirectory, 'OTColors', sorted(colors, key = lambda x: x.key), 'color', createColorGetter)
-    generateComponentsClass(outputDirectory, 'OTComponents', sorted(components, key = lambda x: x.key))
+    if language is 'objc':
+	    generateUITypeClass(outputDirectory, 'OTFonts', sorted(fonts, key = lambda x: x.key), 'font', createFontGetter)
+	    generateUITypeClass(outputDirectory, 'OTColors', sorted(colors, key = lambda x: x.key), 'color', createColorGetter)
+	    generateComponentsClass(outputDirectory, 'OTComponents', sorted(components, key = lambda x: x.key))
+	else		
```

Comment:
As soon this PR is reviewed and adapted this is the place where to start generating the Colors, Fonts and Components Code.